zuul_alpha/core.py

```diff
@@ -85,8 +84,19 @@ def _get_plaintext_private_key(
 
         return response[u'Plaintext']
 
+    class Encryptor:
```

Comment:
I don't like nested classes like this.. can this be pulled out to its own class outside of the Zuul class?